README.md

```diff
@@ -398,13 +412,38 @@ module.exports = {
 }
 ```
 
+## Configuration Object Properties
```

Comment:
Let's be consistent about title capitalization — just the first word and others that require it.